### PR TITLE
Remove setup requirement for the LLVM backend.

### PR DESCRIPTION
**Pull request by lachlansneff**

### CODE REVIEW

**Comment by lachlansneff** [comment]
bors try

File: lib/llvm-backend/Cargo.toml [hunk 1]
```diff
@@ -21,6 +21,9 @@ lazy_static = "1.2.0"
 regex = "1.1.0"
 semver = "0.9"
 rustc_version = "0.2.3"
+reqwest = "0.9.14"
+rust-lzma = "0.4.0"
+tar = "0.4.22"
```
Comment (bjfish) [inline lib/llvm-backend/Cargo.toml]: Could these be build dependencies?

**Comment by lachlansneff** [comment]
bors try

**Comment by lachlansneff** [comment]
bors try

**Comment by lachlansneff** [comment]
bors try

**Comment by ethanfrey** [comment]
Interesting, this seems like it would help dev (I had this issue when first trying to use llvm).
What is the status?

**Comment by syrusakbary** [comment]
Closing this in favor of #1023

**Pull request closed without merging**